internal/controller/dto/user.go

```diff
@@ -0,0 +1,29 @@
+package dto
+
+import (
+	"time"
+
+	"github.com/ivas1ly/gophermart/internal/entity"
+)
+
+type UserResponse struct {
+	ID        string `json:"id"`
+	Username  string `json:"username"`
+	CreatedAt string `json:"created_at"`
+	UpdatedAt string `json:"updated_at"`
+	AuthToken string `json:"auth_token"`
+}
+
+type UserRequest struct {
+	Username string `json:"login" validate:"required,gte=2,lte=255"`
+	Password string `json:"password" validate:"required,gt=8,lte=1000"`
+}
+
+func ToUserFromService(user *entity.User) *UserResponse {
```

Comment:
```suggestion
func ToUserResponse(user *entity.User) *UserResponse {
```
from в целом понятно из аргумента, а то что это именно респонс важней